kep/119-node-resource-topology-aware-scheduling/README.md

```diff
@@ -274,7 +293,7 @@ roleRef:
 
 # Use cases
 
-Numbers of kubernetes worker nodes on bare metal with NUMA topology. TopologyManager feature gate enabled on the nodes. In this configuration, the operator does not want that in the case of an unsatisfactory host topology, it should be re-scheduled for launch, but wants the scheduling to be successful the first time.
+Numbers of kubernetes worker nodes on bare metal with NUMA topology. TopologyManager feature gate enabled on the nodes. In this configuration, the operator does not want that in the case of an unsatisfactory host topology, it should be re-scheduled for launch, but wants the scheduling to be successful the first time. In the case of some available hosts, it should be able to prioritize the most suitable one based on a pre-configured set of instructions.   
```

Comment:
Add "Moreover, " ahead of "in the case of ..."

Reply:
Done. Please let me know before the merge, so I'll squash everything.